packages/vue/src/components/atoms/SfQuantitySelector/SfQuantitySelector.vue

```diff
@@ -59,9 +67,28 @@ export default {
       type: Boolean,
       default: false,
     },
+    /** Minimum allowed quantity */
+    min: {
+      type: Number,
+      required: false,
+      default: null,
+    },
+    /** Maximum allowed quantity */
+    max: {
+      type: Number,
+      required: false,
+      default: null,
+    },
   },
-  handleInput(qty, listeners) {
-    return listeners.input && listeners.input(qty < 1 || isNaN(qty) ? 1 : qty);
+  handleInput(qty, listeners, min, max) {
```

Comment:
Have you tested how does it work when there are no `min` and `max` props passed? This is our case because when we release a new patch version of Storefront UI we would like to avoid breaking changes as much as possible. So in this case you have to answer the question: "Will it brake for someone that have not set min and max props?"

Reply:
Yes, I kept this in mind while testing. The `min` and `max` values default to `null`, so I account for that in the checks, making sure they have a value before using them. I tried to keep the behavior the same as today if min/max are `null`.

I couldn't find a way to reference the props directly from inside this `handleInput` function, which is why I passed them as params from the template. I'm not sure if there's a better approach to this in a functional component.